roles/setup_dbt3/defaults/main.yml

```diff
@@ -1,10 +1,14 @@
 ---
-dbt3_version: "1.11.0"
+dbt3_version: "2.0.1"
 dbt3_pgdata: "/pgdata/dbt3/pgdata"
-pg_version: 14
```

Comment:
If `pg_version` is removed here, I think the variables within the `roles/setup_dbt3/vars/**` files should be reviewed, some unused vars (`pg_data`) use `pg_version`, not sure if this will throw errors while importing the vars but still a good idea to clean up

Reply:
Ah, I don't think the `vars` stuff is needed at all now.  Removing.